solutions/src/main/java/ru/sberbank/school/task02/Converter.java

```diff
@@ -0,0 +1,54 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.exception.FxConversionException;
+import ru.sberbank.school.task02.util.*;
+
+import java.math.BigDecimal;
+import java.util.Comparator;
+import java.util.List;
+
+public class Converter implements FxConversionService {
+
+    private ExternalQuotesService externalQuotes;
+
+    public Converter(ExternalQuotesService externalQuotesService){
```

Comment:
WhitespaceAround: &#39;{&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1105025051>*